data/bios.js

```diff
@@ -9,7 +9,7 @@ export default {
     name: 'Eva Ruth Moravec',
     image: evaRuthMoravecImage,
     bio:
-      '<p>Executive Director and co-founder <a href="https://www.linkedin.com/in/eva-ruth-moravec-m-a-1aa9332/" target="_blank" rel="noopener noreferrer">Eva Ruth Moravec</a> is a 2018<span style="font-weight: 400;"> John Jay/Harry Frank Guggenheim Criminal Justice Reporting fellow, a <a href="https://muckrack.com/profile/portfolio" target="_blank" rel="noopener noreferrer">freelance reporter</a> </span>and the author of a forthcoming book that explores the legality of police shootings in Texas. <span style="font-weight: 400;">While in a data journalism class for her Master’s degree at the University of Texas at Austin, Eva Ruth started a database of officer-involved shootings in Texas. She then explored cases in her database through “<a href="http://pointofimpacttx.com/" target="_blank" rel="noopener noreferrer">Point of Impact</a>,” an investigative journalism series that ran in three Texas daily newspapers. </span><span style="font-weight: 400;">She has covered criminal justice in Texas for a decade, including stints at the </span><i><span style="font-weight: 400;">San Antonio Express-News</span></i><span style="font-weight: 400;"> and The Associated Press. Find her on Twitter <a href="https://twitter.com/EvaRuth?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor" target="_blank" rel="noopener noreferrer">here</a>.</span></p>',
+      '<p>Executive Director and co-founder <a href="https://www.linkedin.com/in/eva-ruth-moravec-m-a-1aa9332/" target="_blank" rel="noopener noreferrer">Eva Ruth Moravec</a> is a 2018<span style="font-weight: 400;"> John Jay/Harry Frank Guggenheim Criminal Justice Reporting fellow, a <a href="https://muckrack.com/profile/portfolio" target="_blank" rel="noopener noreferrer">freelance reporter</a> </span> covering criminal justice in Texas and throughout the U.S. <span style="font-weight: 400;">While in a data journalism class for her Master’s degree at the University of Texas at Austin, Eva Ruth started a database of officer-involved shootings in Texas. She then explored cases in her database through “<a href="http://pointofimpacttx.com/" target="_blank" rel="noopener noreferrer">Point of Impact</a>,” an investigative journalism series that ran in three Texas daily newspapers. </span><span style="font-weight: 400;">She has covered criminal justice in Texas for a decade, including stints at the </span><i><span style="font-weight: 400;">San Antonio Express-News</span></i><span style="font-weight: 400;"> and The Associated Press. Find her on Twitter <a href="https://twitter.com/EvaRuth?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor" target="_blank" rel="noopener noreferrer">here</a>.</span></p>',
```

Comment:
Could we check with @EvaRuthM about where the link for "freelance reporter" should lead to? It looks like a link to the profile of the currently logged-in user rather than Eva's public profile.

Reply:
Hi @nholden, it should go [here](https://muckrack.com/evaruth/portfolio)